CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 5.0.3 (23 August 2022)
+
+Important: Be aware that the docker-container needs CAP_SYS_ADMIN since version 4.7.0
```

Comment:
This probably should be marked as a breaking change, now that you mention it here.. Thus we'd need to bump the version to 6.0.

But then again, it *was* already breaking since 4.7.0, so the corresponging break was already associated with a major version bump. I'd say let's leave it, just wanted to point it out